node/babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
This is usually not required, unless we need very specific custom babel configs, which doesn't seem the case here. Do you know why you had to add this file?

Reply:
I see! I believe I also got this kind of problem before, it's something with the tsconfig configuration not knowing that esmodules should be used. My guess is that if you set `esModuleInterop: true` is node/tsconfig.js it should work without this file. Can you try it out? For the other repo I setup tests for, this line is [there](https://github.com/vtex/buyer-organizations-supplier/blob/main/node/tsconfig.json), so it may be what makes it work there and not here.

Reply:
I put the config but didn't work... :-(

Reply:
Don't worry, then. Let's leave this babel file here if it's working. We'll optimize these setups later, they're really complex right now :(